image/svg/arrow-down.svg

```diff
@@ -1 +1 @@
-<svg alt="Arrow Down" focusable="false" enable-background="new 0 0 10 6" viewBox="0 0 10 6" xmlns="http://www.w3.org/2000/svg"><g fill="currentColor"><path d="m5 5.932c-.222 0-.443-.084-.612-.253l-4.134-4.134c-.338-.338-.338-.886 0-1.224s.886-.338 1.224 0l3.522 3.521 3.523-3.521c.336-.338.886-.338 1.224 0s .337.886-.001 1.224l-4.135 4.134c-.168.169-.39.253-.611.253z"/></g></svg>
\ No newline at end of file
+<svg focusable="false" enable-background="new 0 0 10 6" viewBox="0 0 10 6" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Arrow Down"><title>Arrow Down</title><g fill="currentColor"><path d="m5 5.932c-.222 0-.443-.084-.612-.253l-4.134-4.134c-.338-.338-.338-.886 0-1.224s.886-.338 1.224 0l3.522 3.521 3.523-3.521c.336-.338.886-.338 1.224 0s .337.886-.001 1.224l-4.135 4.134c-.168.169-.39.253-.611.253z"/></g></svg>
```

Comment:
Hmm, `<title>` tags are usually used inside the `<head>` tag to give a web page a title. This is the value we see inside the chrome tab.

For accessibility, I believe we want to use a `title` attribute instead? e.g. `<svg title="Arrow Down">`

Reply:
https://www.w3.org/TR/SVG11/struct.html#DescriptionAndTitleElements

Reply:
I would've though so, but, according to documentation, it needs to be an element inside the SVG tag. Here's 2 more links about doc this subject : https://css-tricks.com/accessible-svgs/ and https://stackoverflow.com/questions/4697100/accessibility-recommended-alt-text-convention-for-svg-and-mathml

Reply:
I see, this looks good then, thanks 💯